src/test/java/pl/touk/sputnik/TestEnvironment.java

```diff
@@ -33,11 +33,11 @@ protected Review review(String filename) {
         return new Review(ImmutableList.of(new ReviewFile(Resources.getResource(filename).getFile())), formatter);
     }
 
-    protected Review nonexistantReview() {
+    protected Review nonExistentReview() {
         return new Review(ImmutableList.of(new ReviewFile("test")), formatter);
     }
 
-    protected Review nonexistantReview(String filename){
+    protected Review nonExistentReview(String filename){
```

Comment:
\[Checkstyle\] ERROR: '\{' is not preceded with whitespace\.

Reply:
Done